hazelcast/src/main/java/com/hazelcast/client/config/ClientConfigXmlGenerator.java

```diff
@@ -352,8 +352,8 @@ private static void serializers(XmlGenerator gen, SerializationConfig serializat
     private static void nativeMemory(XmlGenerator gen, NativeMemoryConfig nativeMemory) {
         gen.open("native-memory", "enabled", nativeMemory.isEnabled(),
                 "allocator-type", nativeMemory.getAllocatorType())
-                .node("size", null, "value", nativeMemory.getSize().getValue(),
-                        "unit", nativeMemory.getSize().getUnit())
+                .node("size", null, "value", nativeMemory.getCapacity().getValue(),
```

Comment:
Are we going to update the XSD as well, introducing a new element for the `capacity`?